_data/hosting_providers.json

```
@@ -2298,5 +2298,15 @@
     "plan": "",
     "reviewed": "2020.01.29",
     "note": "Shared Web Hosting utilizes Plesk Panel interface to enable Let's Encrypt; Managed Cloud Hosting enabled via Certbot on request."
+  },
+    {
+    "name": "CA Net, Europe",
+    "link": "https://www.canet.at",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.canet.at/hosting-server/",
+    "plan": "All plans",
```

```suggestion
    "plan": "",
```
This field is supposed to be a link, but per https://github.com/certbot/website#hosting-providers-list, it won't show up because you have announcement filled out.